### ISSUE TITLE
FATAL: Failed to extract file jdk-8u181-linux-x64.tar.gz! Version 

### ISSUE THREAD

**Issue by Dave-Machado-Bose** [sous-chefs/java#499]
### Cookbook version
2.2.0

### Chef-client version
Chef Client, version 13.11.3

### Platform Details
Ubuntu 16.04.5

### Scenario:
install Oracle Java 8 

### Steps to Reproduce:
I just have a simple "Includes java" in a recipe

**Attributes set as follows:**
default['java']['jdk_version'] = '8'
default['java']['install_flavor'] = 'oracle'
default['java']['oracle']['accept_oracle_download_terms'] = true

### Expected Result:
You will see the following error message: 
Java-8-Oracle will not install and you will receive the following: FATAL: Failed to extract file jdk-8u181-linux-x64.tar.gz error message

### Actual Result:
FATAL: Failed to extract file jdk-8u181-linux-x64.tar.gz

**Comment by espoelstra** [comment]
This is due to Oracle removing old download links when they publish a new version. You can temporarily override by going to the Oracle site and getting the updated link and checksum (or see below). Once somebody submits a PR with the updated information and it gets published to the Supermarket this should resolve itself.

Add this in your wrapper cookbook to temporarily work around the issue.

```
default['java']['jdk']['8']['x86_64']['url'] = 'http://download.oracle.com/otn-pub/java/jdk/8u191-b12/2787e4a523244c269598db4e85c51e0c/jdk-8u191-linux-x64.tar.gz'
default['java']['jdk']['8']['x86_64']['checksum'] = '53c29507e2405a7ffdbba627e6d64856089b094867479edc5ede4105c1da0d65'
```

**Comment by espoelstra** [comment]
Closed by #498 .
